src/main/java/virushade/tasks/TaskList.java

```diff
@@ -0,0 +1,210 @@
+package virushade.tasks;
+
+import virushade.Storage;
+import virushade.StringManipulator;
+import virushade.VirushadeException;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    /**
+     * This is the array that Virushade keeps the tasks in.
+     */
+    private static final ArrayList<Task> TASKS = new ArrayList<>();
```

Comment:
Should this be final? Will TASKS ever change? Sorry if noob question it seems confusing to me

Reply:
I also not sure haha, I made it final cuz IntelliJ hinted at me. I'm not sure what a final ArrayList means though.